README.md

```diff
@@ -483,48 +483,49 @@ It is also possible for a backend to send multiple responses for a
 request or not send any responses for a request. A backend may also
```

Comment:
I think we need to update this to clarify they need to send a response (though it can be empty) for every request.

Reply:
Thank you for the feedback! Perhaps we need to clarify whether or not we are considering the flag as a response?

It sounds like from the verbiage [here](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/decoupled_models.md#knowing-when-a-decoupled-inference-request-is-complete) around clients optionally opting in to receive an empty final response that sending the flag does not always generate a response. So it would not be equivalent to a response in my reading of the documentation.

Reply:
@nnshah1 Is that acceptable? Or would you like to discuss whether or not we are considering signaling the final flag as a response?

Reply:
From the backend side I consider that the responsibility is to send a response with a final flag for every request.
Whether the client recieves that is a seperate concern - does that make sense?

Reply:
Sure! Thanks for clarifying. Let me know whether the new verbiage looks good to you.

Reply:
reading it now, maybe we just remove the second part and leave the original:

"it's possible for a backend to send multiple responses."

That seems to read better - and we just remove the possibility of '0' responses.

I agree its a little confusing between what the backend sends and what the client receives - but this part is primarily for the backend. 

what is available to client is also partly determined by the front end -

Reply:
Thank you for your detailed feedback and explanations! Incorporated it into the latest commit.

Reply:
I think we should properly document what do we mean by the response in the backend's context. It might be a bit confusing. For a longer term goal, if we are treating empty flags-only signal as a response from backend's context,
then we should consider **deprecating** `TRITONBACKEND_ResponseFactorySendFlags` which doesn't consider the SendFlags as an explicit response. We updated the grpc frontend recently to generate a response out of this final flag signal. 

I imagine that if we are treating flags only empty signal as a response from the backend's side, then it should happen with a unified API TRIONBACKEND_ResponseSend. 

@dyastremsky @nnshah1  what are your thoughts?